openedx/core/djangoapps/catalog/tests/test_utils.py

```diff
@@ -154,6 +154,58 @@ def test_get_one(self, mock_warning, _mock_info):
 
 
 @skip_unless_lms
+@ddt.ddt
+class TestGetProgramsWithType(CacheIsolationTestCase):
```

Comment:
This should only subclass `TestCase`. The code under test doesn't use the cache!